pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[tool.black]
+# default is 88
+line-length = 120
```

Comment:
Can we drop this back to 88? 120 I think is good for a linter, but too much for a formatter.

Reply:
It will autoformat every line to 88 though. I fucking hate 88 length limits

Reply:
My complaint is that it's reformatting things to 120 that had been manually formatted to be shorter and more readable.

Reply:
Alright can we compromise at 100?